ide/nanoPG.ml

```diff
@@ -160,6 +160,9 @@ let emacs = insert emacs "Emacs" [] [
   mkE _e "e" "Move to end of line" (Motion(fun s i ->
     (if not i#ends_line then i#forward_to_line_end else i),
     { s with move = None }));
+  mkE ~mods:mM _Right "->" "Move to end of buffer" (Motion(fun s i ->
+    i#forward_to_end,
+    { s with move = None }));
```

Comment:
Implementing "move to beginning of buffer" is just a matter of
```ocaml
Motion (fun s i ->
let buffer = new GText.buffer i#buffer in
buffer#start_iter, ...
)
```

Reply:
Ah thanks, I did not know the `i#buffer`.

Is it worth to do it or would it be better to move the current Ctrl-Home and Ctrl-End to other bindings so as to release the (as-far-as-I-could-understand) already existing bindings of Ctrl-Home and Ctrl-End to the genuine go-to-start and go-to-end?

Reply:
No preferences. This is µPG-specific anyways, right?

Reply:
Yes, but if Ctrl-Home and Ctrl-End are eventually supported even w/o µPG (nPG?), there is no point in giving another binding in µPG. Moreover, if Ctrl-Home and Ctrl-End are eventually supported and Ctrl-Alt-Home and Ctrl-Alt-End are used instead for Restart-Coq and Eval-to-End, we shall have to invent yet new bindings for Restart-Coq and Eval-to-End under MacOS X since Ctrl-Alt is not available under MacOS and Ctrl-Cmd-Home and Ctrl-Cmd-End will already be used for µPG.

Shortly: Having two distinct variants of Home and End in µPG and not in µPG would seem overkill.

Until a clear-cut conclusion goes out of #9899, I however added a `Meta-Left` for go-to-start. If a variant is proposed outside μPG, the ones in μPG will be removable.

Other question: I think that keeping the name nanoPG is preferable (rather than keeping the name μPG), because the name nanoPG is hard-wired in the preference file. No objection if I backtrack on the uniformization nanoPG -> μPG to do it instead the other way round?

Reply:
It doesn't really matter that there is a discrepancy in the preference file entry, this is not supposed to be read by the end user anyways. It could as well be some garbage identifier.

Reply:
Yes, but it is generally clearer, at least for debugging, when feature A is called A and not B. Anyway, you seem to prefer keeping the name μPG, so I'll keep the name nanoPG only in the preference file (for compatibility).